static/src/stylesheets/module/journalism/_inline-audio-player.scss

```diff
@@ -0,0 +1,101 @@
+.inline-audio_container {
```

Comment:
You can also add [`contain: strict`](https://developer.mozilla.org/en-US/docs/Web/CSS/contain). This is because you are doing a lot of mutations inside the player and we don't want them to reverberate across the whole page.

Reply:
(I would fix the height to `100px` too, for good measure)

Reply:
Will go for `min-height: 100px` - the container actually needs to get slightly taller on small mobiles